docs/topology/usage.rst

```diff
@@ -142,6 +142,88 @@ This is a convenience function, as under the hood it uses threads perform
 per device ``.connect()`` asynchronously. All other behavior follows that of
 the above connection manager concept. 
 
+Disconnect From All Devices
+----------------------
+
+``Testbed`` object also provides a convenience function, ``Testbed.disconnect()``,
+allowing you to make asynchronous disconnection from multiple testbed devices
+at the same time.
+
+.. code-block:: python
+    # Example
+    # -------
+    #
+    #   disconnecting from devices in parallel
+    # using the sample topology file from
+    from pyats import topology
+    testbedfile = os.path.join('sampleTestbed.yaml')
```

Comment:
keep it simple, just say

    testbed = topology.loader.load('your-testbed-file.yaml')

you arent joining a path anyway